docs/Using-Fleet/REST-API.md

```diff
@@ -2586,7 +2586,7 @@ Deletes the query specified by name.
 
 #### Example
 
-`DELETE /api/v1/fleet/queries/{name}`
+`DELETE /api/v1/fleet/queries/query_title`
```

Comment:
```suggestion
`DELETE /api/v1/fleet/queries/my_query`
```

Great catch 💯  @DominusKelvin what do you think about using a different example query name (something like `my_query` which makes it obvious that this is an example URL?

I think Mike, and other users, might be confused when they read `query_title` because it reads like a dynamic variable name.

Reply:
Sure